routes/users.js

```diff
@@ -511,3 +511,18 @@ router.get('/dashboard1/:username/myapplications', function(req, res){
 router.get('/calendar', loggedIn , function(req, res){
 	res.render('calendar');
 });
+
+router.post('/get-maildata',function(req, res){
+	var mail = req.body.mail;
+	var res = req.body.data;
```

Comment:
I think variable 'res' name should change because res is a defined parameter in the  function.